Files/LUISApp.json

```diff
@@ -142,17 +142,17 @@
       "entities": []
     },
     {
-      "text": "list new idea",
```

Comment:
This is not a typo. It was specially created for better compatibility in case that users may misspell it.
You can find "list new ideas" on [line 170](https://github.com/OfficeDev/Office-365-Huddle-Templates/blob/master/Files/LUISApp.json#L170)

Reply:
Reverted